src/AGCertificateVerifier.h

```diff
@@ -189,11 +264,11 @@ class AGCertificateVerifier {
 
     AGVerifyResult verifyPins(const std::string &dnsName, STACK_OF(X509) *certChain);
 
-    AGVerifyResult verifyRevocations(STACK_OF(X509) *certChain);
+    AGVerifyResult verifyCrlSetsStatus(STACK_OF(X509) *certChain);
 
     AGVerifyResult verifyUntrustedAuthority(STACK_OF(X509) *certChain);
```

Comment:
Still unclear what certificates are untrusted, it should be explicitly described